src/main/java/org/opensearch/flowframework/model/ResourceCreated.java

```diff
@@ -47,6 +50,14 @@ public class ResourceCreated implements ToXContentObject, Writeable {
     public ResourceCreated(String workflowStepName, String workflowStepId, String resourceId) {
         this.workflowStepName = workflowStepName;
         this.workflowStepId = workflowStepId;
+        this.resourceType = WorkflowResources.getResourceByWorkflowStep(workflowStepName);
```

Comment:
Should we maybe change the code in FlowFrameworkIndicesHandler to pass this WorkflowResources.getResourceByWorkflowStep(workflowStepName) when creating the `ResourcesCreated` object so the class here is more generic?

Reply:
good idea, ill make this change